### DISCUSSION TITLE
Project Name

### DISCUSSION THREAD

**Discussion by detronetdip** [detronetdip/tello-server#1] [Ideas]
We should give this project a meaningful name. Currently I have given a name `SPEAKER`, it is just temporary, so I want you all(@parijat965 and @Sinchan26)  to suggest a meaningful name for our project.

**Comment by Sinchan26** [comment]
talk-to-match  (sounds like talk-to-much)

**Reply by detronetdip** [reply]
While it is a good name as we usually do talk to know each other in general  but still it is sounding like some dating app. As we are gonna build a community platform so I think names which are intended to be a bit personal should not be used.

**Comment by parijat965** [comment]
I think Mediocity sounds good

**Reply by detronetdip** [reply]
We can keep this on note and with that let's find more.

**Comment by parijat965** [comment]
I was thinking about the name `broad.cast`. Let me know if we can proceed with this.

**Reply by detronetdip** [reply]
@parijat965  it is a good one. I think we can go with this. @Sinchan26 what about you?

**Comment by detronetdip** [comment]
We will be continuing with the name `tello`.
